gui/qt/i18n/nl_NL.ts

```diff
@@ -312,12 +312,12 @@
     <message>
         <location filename="../mainwindow.ui" line="3337"/>
         <source>Equates:</source>
-        <translation>Equates:</translation>
+        <translation>Staat gelijk aan:</translation>
```

Comment:
I think just the English translation is better IMHO.

Reply:
Agreed. How do I do this? GitHub is pretty new to me.
Do I need to Split?